comms/email.md

```diff
@@ -0,0 +1,13 @@
+Hello everyone,
+
+We're looking forward to seeing you all at the What They Forgot to Teach You About R workshop at {LOCATION} on {DATES}. This email covers important setup that you'll need to do before the workshop.
+
+To fully participate in the workshop, you'll need to bring with you a laptop you can install R packages on and that has:
+
+1. A recent version of R (we recommend 3.6+)
+2. A recent version of RStudio (we recommend 1.2.5000+; versions earlier than 1.1 will _not_ work)
+3. Git and GitHub set up and connected to RStudio
+
+Instructions on the necessary git and GitHub setup are here: https://happygitwithr.com/workshops.html#pre-workshop-set-up. Completing this will take at least 1-2 hours, and really does need to be done before the workshop starts.
+
+We are happy to help if you run into any issues. Please post questions on {COMMUNITY THREAD} and one of us will get back to you there.
```

Comment:
We will also have TAs onsite who may be able to help during a break prior to the first git/GitHub session, but do not rely on that. We can't deal with many people or with very tricky issues in that setting. It is much better to complete the setup in advance.